cmd/api/handlers/player/get.go

```diff
@@ -0,0 +1,33 @@
+package player
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/jairogloz/go-l/cmd/api/core"
+	"github.com/jairogloz/go-l/internal/domain"
+)
+
+// GetPlayer godoc
+// @Summary Get a player
+// @Description get a player with id param
+// @Tags players
+// @Accept  json
+// @Produce  json
+// @Param PlayerIdParam body core.PlayerIdParam true "Create player"
+// @Success 200 {object} map[string]interface{} "player: domain.Player"
+// @Failure 400 {object} map[string]interface{} "error: string"
+// @Router /players/:id [get]
+func (h Handler) GetPlayer(c *gin.Context) {
+	var playerIdParam core.PlayerIdParam
+	if err := c.ShouldBindUri(&playerIdParam); err != nil {
+		core.RespondError(c, domain.NewAppError(domain.ErrCodeInvalidParams, err.Error()))
+		return
+	}
+
+	player, err := h.PlayerService.Get(playerIdParam.Id)
```

Comment:
Como sugirió @DarcoProgramador, es más sencillo usar `c.Param("<paramName>")` y así nos ahorramos un el type PlayerIdParam, para luego no sobrepoblar de types 😉 

```suggestion
	playerIdParam := c.Param("id")

	player, err := h.PlayerService.Get(playerIdParam)
```